core/src/ragtime/core.clj

```diff
@@ -53,12 +53,16 @@
   ([store index migrations options]
    (let [strategy (:strategy options strategy/raise-error)
          reporter (:reporter options reporter/silent)
-         index    (into-index index migrations)
-         applied  (p/applied-migration-ids store)]
+         index (into-index index migrations)
+         applied (p/applied-migration-ids store)]
```

Comment:
The indentation of these lines has changed. Indentation changes are not relevant to the feature, so should be removed.